packages/now-build-utils/src/types.ts

```diff
@@ -320,3 +321,50 @@ export interface BuilderFunctions {
     excludeFiles?: string;
   };
 }
+
+export interface NowRewrite {
```

Comment:
Instead of defining these types again, can they be imported from `@now/routing-utils`?

Reply:
I was thinking about that, but then it means that we have to make `@now/routing-utils` a regular dependency of `@now/build-utils`, which seemed undesirable to me. I can switch it if you think it's ok though.